<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraParaClass</class>
 <widget class="QWidget" name="cameraParaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cameraPara</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="labelExposure">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>曝光时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGain">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>增益：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDepthScale">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>深度范围：</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAutoExposure">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>开启自动曝光</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="numDepthMin">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="numDepthMax">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>500.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="numExposure">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="value">
       <double>10000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="numGain">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
